test/msd_test.py

```diff
@@ -296,6 +296,11 @@ def test_mass_storage(workspace, parent_test):
     test.set_flush_size(0x1000)
     test.run()
 
+    if target.name == "BlueNinja":
+        test_info.info("Skip the remaining MSD test.")
+        test_info.info("Writing incomplete binary to TZ10xx MCU, CoreSight becomes unresponsive.")
```

Comment:
Providing a Windows 7 environment to identify the critical test items ran the test script.
(In Windows 10 Pro execution of the test script was unstable ...)

Was "Load blank binary + vector table" only is there is no response of CoreSight result.
Other test was able to run without problems.

It fixes to only skip the "Load blank binary + vector table".